components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -339,6 +349,32 @@ class AddonsManagerAdapter(
             }
         }
     }
+
+    fun refresh(addon: Addon){
+        addonsMap[addon.id] = addon
+        submitList(createListWithSections(addonsMap.values.toList()))
+    }
+
+    fun refresh(addons: List<Addon>){
+        addonsMap = addons.associateBy({ it.id }, { it }).toMutableMap()
+        submitList(createListWithSections(addons))
+    }
+
+    private object DifferCallback : DiffUtil.ItemCallback<Any>() {
```

Comment:
This help us to create a diff of between the current list and the list that we want to update.